### DISCUSSION TITLE
I need help installing SEconomy

### DISCUSSION THREAD

**Discussion by Voidity** [Pryaxis/TShock#2195] [Q&A] [answered]
I've found https://github.com/Quinci135/SEconomy but don't know how to install or use any of it.
I am not sure if it is possible or not but if it IS possible can someone please help me?
Thanks in advance.

**Comment by Kojirremer** [comment, marked as answer]
I'm not willing to go as far as helping with configuration of the plugins, but if it's just installing, you just need to toss all DLL files [here](https://github.com/Quinci135/SEconomy/tree/master/Builds) to the ServerPlugins folder, and then run the server once for it to create its config files on `/tshock/SEconomy/`
